simple_history/models.py

```diff
@@ -4,9 +4,21 @@
 import warnings
 from dataclasses import dataclass
 from functools import partial
-from typing import TYPE_CHECKING, Any, Dict, Iterable, List, Sequence, Type, Union
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    ClassVar,
+    Dict,
+    Iterable,
+    List,
+    Sequence,
+    Type,
+    Union,
+)
 
 import django
+from asgiref.local import Local
```

Comment:
You don't have to change to it, but why not a [`ContextVar`](https://docs.python.org/3/library/contextvars.html)?